src/static/templateUrl.ts

```diff
@@ -86,11 +86,21 @@ export default class TemplateURL extends URL {
     template: string,
     parameters?: PathParameters
   ): string {
-    return parameters
+    let templatedUrl = parameters
       ? template.replace(
           /\{([^\}]+)\}/g /* eslint-disable-line no-useless-escape */,
           (match, param: string) => String(parameters[param])
         )
       : template;
+
+    // Regex for ./ ../ and encoded variants
+    const regex = /(\.|%2e|%2E|%252e|%252E)+(\/|%2f|%2F|%252f|%252F)+/g;
```

Comment:
[NIT] rename this to be more specific like `pathRegex` or `pathTraversalRegex`

Reply:
This change has been applied